rust/operator-binary/src/config/jvm.rs

```diff
@@ -123,8 +123,10 @@ fn recommended_trino_jvm_args(product_version: &str) -> Result<Vec<String>, Erro
             "-XX:+UnlockDiagnosticVMOptions".to_owned(),
             "-XX:G1NumCollectionsKeepPinned=10000000".to_owned(),
         ]),
-        // Copied from https://trino.io/docs/455/installation/deployment.html#jvm-config
-        "455" => Ok(vec![
+        // Copied from:
+        // - https://trino.io/docs/455/installation/deployment.html#jvm-config
+        // - https://trino.io/docs/469/installation/deployment.html#jvm-config
+        "455" | "469" => Ok(vec![
```

Comment:
```suggestion
        "455" | "470" => Ok(vec![
```

Reply:
This has been changed to support numeric ranges.
See: 264042614e91b2212ee125782dffd4bb6e9baf73

Reply:
Nice, thanks :)